springMicroblogContinued/src/main/java/com/theironyard/Message.java

```diff
@@ -0,0 +1,42 @@
+package com.theironyard;
+
+import javax.persistence.*;
+
+/**
+ * Created by Nigel on 7/19/16.
+ */
+
+@Entity
+@Table(name="microblog")
```

Comment:
This should reflect the object you are storing.  `microblog` is more descriptive of the application.  This should be `messages`